src/Okta.DNX.OAuth.ResourceServer/Startup.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+using Okta.DNX.OAuth.ResourceServer.Models;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Protocols;
+using System.Security.Claims;
+
+namespace Okta.DNX.OAuth.ResourceServer
+{
+    public class Startup
+    {
+        string clientId = string.Empty;
```

Comment:
these fields are initialized in constructor only, so, we could make it `readonly`